agent/lib/kontena/workers/container_log_worker.rb

```diff
@@ -67,7 +68,7 @@ def on_message(id, stream, chunk)
         type: stream,
         data: data
       }
-      @queue << msg
+      @queue << msg if @queue.size < QUEUE_LIMIT
```

Comment:
Maybe we should log only when agent hits this limit.. and then log again when queue can be used? Actually this might be better logic anyway..  we probably should not start to fill queue again if it's almost full.